src/main/java/core/basesyntax/service/impl/ReportCreatorImpl.java

```diff
@@ -0,0 +1,23 @@
+package core.basesyntax.service.impl;
+
+import core.basesyntax.db.Storage;
+import core.basesyntax.service.ReportCreator;
+import java.util.Map;
+
+public class ReportCreatorImpl implements ReportCreator {
+    private static final String SEPARATOR = ",";
+    private static final String HEADER = "fruit,quantity";
+
+    @Override
+    public String createReport() {
+        StringBuilder stringBuilder = new StringBuilder(HEADER);
```

Comment:
list list, stringbuilder stringbuilder are bad naming examples, it doesn't tell us anything about what it actually stores